oauth2/oauth2-resource-server/src/main/java/org/springframework/security/oauth2/server/resource/authentication/JwtReactiveAuthenticationManager.java

```diff
@@ -74,6 +78,9 @@ public void setJwtAuthenticationConverter(
 	}
 
 	private AuthenticationException onError(JwtException e) {
+		if (logger.isDebugEnabled()) {
```

Comment:
I'm not seeing the added benefit of this one since `AuthenticationWebFilter` displays the same information. Is it necessary?

Reply:
Agree. I've removed